spec/segment/analytics/e2e_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+
+module Segment
+  # End-to-end tests that send events to a segment source and verifies that a
+  # webhook connected to the source (configured manually via the app) is able
+  # to receive the data sent by this library.
+  describe 'End-to-end tests', e2e: true do
+    WRITE_KEY = 'qhdMksLsQTi9MES3CHyzsWRRt4ub5VM6'
```

Comment:
Could you add a comment that this writeKey corresponds to https://app.segment.com/segment-libraries/sources/analytics_ruby_e2e_test/overview